sign/eddilithium3/eddilithium_test.go

```diff
@@ -12,7 +12,7 @@ func BenchmarkVerify(b *testing.B) {
 	// instead of at the moment of verification (as compared to the reference
 	// implementation.  A fair comparison thus should sum verification
 	// times with unpacking times.)
-	var seed [32]byte
+	var seed [57]byte
```

Comment:
Seed is 32 byes for all modes, isn't it?

Reply:
Yes, but Ed448 requires a larger seed.